Lib/test/test_uuid.py

```diff
@@ -591,13 +584,17 @@ def test_unix_getnode(self):
             node = self.uuid._unix_getnode()
         except TypeError:
             self.skipTest('requires uuid_generate_time')
-        self.check_node(node, 'unix')
+        # Since we don't know the provenance of the MAC address, don't check
+        # whether it is locally or universally administered.
+        self.check_node(node, 'unix', admin=False)
```

Comment:
I would prefer to not do non-guarantied tests by default, and do them only if this is specified explicitly.

Reply:
Hmm, I obviously mildly (+0) prefer it the other way around.  Would your opinion change if we named the flag `universal`?

Reply:
This is your check, I left it on to you.

Reply:
I chose `check_bit` as the argument name, but didn't change the default value.  I'm happy with this.